### PR TITLE
Create 3420-10-3-Marcus-Jones.md +3420-9-29-Lyle-Stevensen.md

### PR DESCRIPTION
**Pull request by NMS-Seventh-Fleet**
Added next blog entry.

### CODE REVIEW

File: _posts/3420-10-3-Marcus-Jones.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+![Image of Animal 2 on Eithon]
+(https://nms-seventh-fleet.github.io/images/image1.png)
+
+![Image of Animal 3 on Eithon]
+(https://nms-seventh-fleet.github.io/images/image9.png)
+
+<p>On another planet, we found an enormous structure that served as a historical archive for the native Vy'keen.</p>
+
+![Image 1 of historical archive in the Vodanda-Ilib XII system ]
+(https://nms-seventh-fleet.github.io/images/image9.png)
```
Comment (mray90) [inline _posts/3420-10-3-Marcus-Jones.md]: image9.png is the "Animal 3" image. In fact, can we come up with a better naming scheme for image files? Maybe name them after the post (e.g. jones_3420-10-03_001.png)

File: _posts/3420-10-3-Marcus-Jones.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<p>After some negotiation, they allowed us ten minutes of access to the main data core. I was hoping to dredge up some long-lost portal lore.</p>
+
+![Image 1 of historical archive in the Vodanda-Ilib XII system ]
```
Comment (mray90) [inline _posts/3420-10-3-Marcus-Jones.md]: Should be "Image 2 of..."

File: _posts/3420-10-3-Marcus-Jones.md (hunk 1 above)
Comment (NMS-Seventh-Fleet) [reply _posts/3420-10-3-Marcus-Jones.md]: resolved

File: _posts/3420-10-3-Marcus-Jones.md (hunk 2 above)
Comment (NMS-Seventh-Fleet) [reply _posts/3420-10-3-Marcus-Jones.md]: resolved

**Comment by mray90** [comment]
Looks good to me. Merging now.

**Comment by mray90** [comment]
Looks pretty good, merging.

**Pull request merged**